### DISCUSSION TITLE
Inner Join

### DISCUSSION THREAD

**Discussion by lyudad** [prisma/prisma#17435] [Q&A]
Hello I need to make request like

SELECT * FROM "A" INNER JOIN "B" ON "A".id = "B".a_id;

I have tried a lot of options, but Prisma all the time generates LEFT JOIN relationship. Google and documentation have no information about such cases
I stuck on it for almost 4 days, will appreciate any suggestions

**Comment by arthur-demenskiy** [comment]
same problem!

**Comment by nurul3101** [comment]
Hi @lyudad 👋 
Same response as [here](https://github.com/prisma/prisma/discussions/17436#discussioncomment-4731650).
What data do you want to get with your query? Can you share your schema file and your use case?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
